### DISCUSSION TITLE
Disable focusing IDE/Code Editor after adding a Signal

### DISCUSSION THREAD

**Discussion by yosimba2000** [godotengine/godot-proposals#10187] [Editor]
Win10 x64 22H2
Godot 4.2 .Net
Visual Studio Code

After adding a signal to a node, it always focuses the IDE.
It's pretty annoying to bring Godot back into focus all the time. Can we get this fixed?

**Comment by AThousandShips** [comment]
Do you mean when connecting to a signal?

If you mean moving to the code editor/IDE when connecting to an already existing signal that's already implemented in 4.3, see:
* https://github.com/godotengine/godot/pull/89249
